easypy/logging.py

```diff
@@ -214,7 +214,7 @@ def configure_contextual_logging(_ctx=ExitStack(), **kw):
     _ctx.enter_context(THREAD_LOGGING_CONTEXT(indentation=indentation, **kw))
 
 
-THREAD_LOGGING_CONTEXT = ThreadContexts(counters="indentation", stacks="context")
+THREAD_LOGGING_CONTEXT = ThreadContexts(counters="indentation", stacks="context", defaults=dict(host=''))
```

Comment:
I also think `host` is too specific. It's very awkward doing:
```MultiObject([cat, dog], log_ctx=lambda p: dict(host=p.name).talk()```
cant find a better name from the top of my head but I do think we should change that in a later iteration.

Reply:
Since the parameter name is `log_ctx`, why not use `ctx`, `ctx_name` or `ctx_display_name`?

Reply:
> I also think `host` is too specific. It's very awkward doing:
> `MultiObject([cat, dog], log_ctx=lambda p: dict(host=p.name).talk()`
> cant find a better name from the top of my head but I do think we should change that in a later iteration.

The commit here solves the bug we've had at weka where sometimes the logging would complain about a missing `host` key.
This will change in the refactored weka-agnostic logging I'm working on, allowing the user to define this key. At Vast I called it `domain`.

Anyway - your comments are opening up new issues - the point with the PR is to merge everything into master so we have a solid reference point.
Let's keep these good comments for other PRs and get this one in( the fun thing is to see it auto-publish to pypi).